code401Challenges/src/main/java/code401Challenges/graph/DepthFirst.java

```diff
@@ -0,0 +1,28 @@
+package code401Challenges.graph;
+
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+
+public class DepthFirst {
+    Graph graph = new Graph();
+    Node<String> node;
+
+    // what determines the starting node?
+
+    public List<String> depthFirst (Graph graph) {
```

Comment:
Since a graph has no clear starting point like a list or tree, you can add the starting node as a parameter to the method. Also, if you want to make this method more generic, your method signature would look something like: `public <T> List<T> depthFirst (Graph<T> graph, Node<T> n)`. If you want to learn more about that, google "java generic return type"